src/utils/articleTextSize.js

```diff
@@ -1,8 +1,6 @@
 const KEY = 'article-textsize'
 const DEFAULT_SIZE = 2
 const [MIN_SIZE, MAX_SIZE] = [0, 6]
-const SELECTORS = '.adjustable-font-size'
-const FONT_SIZE_ATTRIBUTE = 'data-font-size'
```

Comment:
here're some context for you, this attribute is used for **telling the element the initial font size** because every element has different font size in place.

so not just a single `font-size-n` to deal with all elements in article.

Reply:
Right, so I'm proposing we do something slightly different now and take that logic out of attributes and into fontsize.less. With CSS we can adjust the font size as needed for different elements with a combination of `font-size-n` and the corresponding other class:`title`, `desc`, and so on. The `@standard` list in this case is the regular font size used in most cases when `font-size-n` stands alone. The mapping is:


class name | standard (px) | title (px) | preview-title (px) | desc (px)
-- | -- | -- | -- | --
font-size-1 | 14 | 22 | 15 | 12
font-size-2 | 15 | 23 | 16 | 13
font-size-3 | 16 | 24 | 17 | 14
font-size-4 | 17 | 25 | 18 | 15
font-size-5 | 18 | 26 | 19 | 16
font-size-6 | 19 | 27 | 20 | 17
font-size-7 | 20 | 28 | 21 | 18

What do you think?